src/ItemBlacklist.Core/ItemBlacklist.UI.cs

```diff
@@ -55,6 +57,42 @@ protected void InitUI()
             scrollRect.movementType = ScrollRect.MovementType.Clamped;
             scrollRect.GetComponent<Image>().color = RowColor;
 
+            {
+                var contentItem = UIUtility.CreatePanel("BlacklistContent", scrollRect.content.transform);
+                contentItem.gameObject.AddComponent<LayoutElement>().preferredHeight = PanelHeight;
+                contentItem.gameObject.AddComponent<Mask>();
+                contentItem.color = RowColor;
+
+                var itemPanel = UIUtility.CreatePanel("BlacklistPanel", contentItem.transform);
+                itemPanel.color = RowColor;
+                itemPanel.gameObject.AddComponent<CanvasGroup>();
+                itemPanel.gameObject.AddComponent<HorizontalLayoutGroup>().padding = Padding;
+
+                FavoriteButton = UIUtility.CreateButton("BlacklistButton", itemPanel.transform, "Favorite this item");
+                FavoriteButton.gameObject.AddComponent<LayoutElement>();
+
+                var text = FavoriteButton.GetComponentInChildren<Text>();
+                text.resizeTextForBestFit = false;
+                text.fontSize = 26;
+            }
+            {
+                var contentItem = UIUtility.CreatePanel("BlacklistModContent", scrollRect.content.transform);
+                contentItem.gameObject.AddComponent<LayoutElement>().preferredHeight = PanelHeight;
+                contentItem.gameObject.AddComponent<Mask>();
+                contentItem.color = RowColor;
+
+                var itemPanel = UIUtility.CreatePanel("BlacklistModPanel", contentItem.transform);
+                itemPanel.color = RowColor;
+                itemPanel.gameObject.AddComponent<CanvasGroup>();
+                itemPanel.gameObject.AddComponent<HorizontalLayoutGroup>().padding = Padding;
+
+                FavoriteModButton = UIUtility.CreateButton("BlacklistModButton", itemPanel.transform, "Favorite all items from this mod");
```

Comment:
Why call these controls "Blacklist..." and not "Favorite..."?

Reply:
I wasn't sure if the string had a special function, fixed now